TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpRequestHandler.java

```diff
@@ -126,6 +131,12 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
     }
 
     protected void channelFinishedReadingAnHttpMessage(ChannelHandlerContext ctx, Object msg, HttpRequest httpRequest) throws Exception {
+        MDC.put("channelId",ctx.channel().id().asLongText());
+        MDC.put("httpMethod", httpRequest.method().toString());
+        MDC.put("httpEndpoint", httpRequest.uri());
+        metricsLogger.info("Full request received");
```

Comment:
I'd like to see the # of bytes that came in.  I'm not sure how easy it is to recover that here (we just need to accumulate the count from the ByteBufs though).